README.md

```diff
@@ -1,6 +1,12 @@
 # Universal Robots ROS2 Driver
+Universal Robots have become a dominant supplier of lightweight, robotic manipulators for industry, as well as for scientific research and education.
+<center><img src="ur_robot_driver/doc/initial_setup_images/e-Series.jpg" alt="Universal Robot e-Series family" style="width: 80%;"/></center>
 
-Beta version of the ROS2 Universal Robots driver. Should be transferred to the Universal Robots org when ready.
+This is one of the very first ROS2 manipulator drivers. Some of the new features are enabled by ROS2 and includes decreased latency, improved security, and more flexibility regarding middleware configuration. Launch files to get started quickly and easily with MoveIt2 are included.
```

Comment:
```suggestion
This is one of the very first ROS2 manipulator drivers. Some of the new features are enabled by ROS2 and include decreased latency, improved security, and more flexibility regarding middleware configuration. Launch files to get started quickly and easily with MoveIt2 are included.
```

Reply:
I think the last part concerning the launch files could misleadingly  suggest that it is necessary to use MoveIt2 to run the driver.

Reply:
I think it is okay to highlight that you easy can get started. but your point is relevant at well.
@MGBla Do you have a suggestion to what we could add or adjust to address this potential mislead?

Reply:
see updated suggestion